integration/ui/satellite/e2e_onboarding_test.go

```diff
@@ -0,0 +1,222 @@
+		assert.Equal(t, "Backwards S3-Compatible API for uploading data programatically.", gatewayText)
+		gatewayButton := page.MustElementX("(//*[@class=\"label\"])[2]").MustText()
+		assert.Equal(t, "Continue", gatewayButton)
+		uplinkBanner := page.MustElementX("(//*[@class=\"overview-area__label\"])[1]").MustText()
+		assert.Equal(t, "End-to-End Encrypted", uplinkBanner)
+		uplinkHeader := page.MustElementX("(//*[@class=\"overview-area__path-section__title\"])[2]").MustText()
+		assert.Equal(t, "Uplink CLI", uplinkHeader)
+		uplinkText := page.MustElementX("(//*[@class=\"overview-area__path-section__text\"])[2]").MustText()
+		assert.Equal(t, "Natively installed client for interacting with the Storj Network.", uplinkText)
+		uplinkButton := page.MustElementX("(//*[@class=\"label\"])[3]").MustText()
+		assert.Equal(t, "Continue", uplinkButton)
+		rcloneBanner := page.MustElementX("(//*[@class=\"overview-area__label\"])[2]").MustText()
+		assert.Equal(t, "End-to-End Encrypted", rcloneBanner)
+		rcloneHeader := page.MustElementX("(//*[@class=\"overview-area__path-section__title\"])[3]").MustText()
+		assert.Equal(t, "Sync with Rclone", rcloneHeader)
+		rcloneText := page.MustElementX("(//*[@class=\"overview-area__path-section__text\"])[3]").MustText()
+		assert.Equal(t, "Map your filesystem to the decentralized cloud.", rcloneText)
+		rcloneButton := page.MustElement(".overview-area__path-section__button")
+		assert.Equal(t, "Continue", rcloneButton.MustText())
+		rcloneLink := *page.MustElement("a.overview-area__path-section__button").MustAttribute("href")
+		assert.Equal(t, "https://docs.storj.io/how-tos/sync-files-with-rclone", rcloneLink)
+
+		// continue to next page
+		page.MustElement("div.overview-area__path-section__button.container.blue-white").MustClick()
+
+		// create an access grant screen - check elements
+		createHeader := page.MustElement("h1.onboarding-access__title").MustText()
+		assert.DeepEqual(t, createHeader, "Create an Access Grant")
+		createText := page.MustElement("p.onboarding-access__sub-title").MustText()
+		assert.DeepEqual(t, createText, "Access Grants are keys that allow access to upload, delete, and view your project’s data.")
+		progressbar := page.MustElement("div.progress-bar").MustVisible()
+		assert.True(t, progressbar)
+		createName := page.MustElement("h1.name-step__title").MustText()
+		assert.DeepEqual(t, createName, "Name Your Access Grant")
+		nameText := page.MustElement("p.name-step__sub-title").MustText()
+		assert.DeepEqual(t, nameText, "Enter a name for your new Access grant to get started.")
+		accessTitle := page.MustElement("h3.label-container__main__label").MustText()
+		assert.DeepEqual(t, accessTitle, "Access Grant Name")
+		accessInput := *page.MustElement("input.headered-input").MustAttribute("placeholder")
+		assert.Equal(t, accessInput, "Enter a name here...")
+		nextButton := page.MustElement("div.container").MustText()
+		assert.DeepEqual(t, nextButton, "Next")
```

Comment:
I'm not sure whether it's worthwhile to check every single element in the ui. For example, reordering things or slightly changing the text would break these tests.